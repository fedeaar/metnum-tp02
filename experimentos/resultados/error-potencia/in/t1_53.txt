2.480885354268882281e+02 9.096209822233516817e+00 4.590692305932626027e+01 -5.219121609027843789e+01 -5.417710187705665703e+01 -1.511423114683300071e+02 -4.101699003411867750e+01 -5.747012584607784902e+01 -1.258519486749578320e+01 -4.782374204528093031e+01 -1.220851252727313039e+02 -2.260831179294159767e+02 -3.589850171074235874e+01 -7.908401864627394140e+00 4.805353425506695864e+00 1.030098176648417336e+02 -4.904375289220831746e+01 6.615952122195514562e+01 3.615497669499554689e+01 3.446143441964952814e+00
9.096209822233522146e+00 -3.208123025251200602e+02 4.794776054588795233e+01 4.064213076975629946e+00 2.392220125866039737e+01 -3.235764976539883264e+00 -1.842134180118596642e+00 8.784021449319272534e+00 1.913451408033145995e+01 3.966086305760949671e+01 2.669867825201457023e+01 -1.982703997041474508e+01 1.805396881141144405e+01 1.307010185173683725e+00 7.945232914620119935e+00 1.128855664933924743e+02 2.361353581390678258e+01 6.271107053896311356e+01 3.351598550397287823e+01 3.893177216325346279e+01
4.590692305932624606e+01 4.794776054588794523e+01 -6.156370776759415548e+02 1.773033749334069853e+01 5.204213722411815013e+01 2.572886699945617295e+01 5.336555180880863780e+00 2.687946268478824763e+01 3.526840277210941110e+01 7.759713262590182126e+01 7.082396683255424819e+01 1.290068040521129689e+01 3.824439568888663388e+01 3.866585077563997608e+00 1.257030230621269595e+01 1.714158318093651303e+02 5.045372622813156482e+01 9.337876022171904822e+01 4.974177081927150113e+01 6.575096023479690643e+01
-5.219121609027843789e+01 4.064213076975629058e+00 1.773033749334069498e+01 3.544820522986519791e+02 -1.867106258662268203e+01 -5.404675506698802678e+01 -1.469418736137630255e+01 -2.027035450050597021e+01 -3.957204716112934673e+00 -1.596245606488100677e+01 -4.283578585356516299e+01 -8.126420990153069113e+01 -1.231004996488575109e+01 -2.786631529547719754e+00 1.937846753166918123e+00 3.993306823721567866e+01 -1.684715153795044529e+01 2.537352310098676611e+01 1.384505174727032717e+01 2.319936547352003942e+00
-5.417710187705667124e+01 2.392220125866038671e+01 5.204213722411815013e+01 -1.867106258662268203e+01 9.524005710182150608e+01 -6.444391951867767432e+01 -1.815523935606591976e+01 -1.758079277909715898e+01 8.048510627347919666e+00 7.735608092681065528e+00 -3.178995946386082849e+01 -1.067580301778642990e+02 -2.293761795812063298e+00 -2.351350146382029216e+00 7.470576060251922890e+00 1.204380623061663300e+02 -3.860557029686396024e+00 7.058402417947785068e+01 3.805129393576854824e+01 2.833314830332178857e+01
-1.511423114683300355e+02 -3.235764976539895699e+00 2.572886699945617295e+01 -5.404675506698802678e+01 -6.444391951867767432e+01 3.908361704344045506e+02 -4.034337098293227086e+01 -6.133728485575400668e+01 -2.071765595501261004e+01 -6.473648773252223521e+01 -1.337030526133585795e+02 -2.182753366558432049e+02 -4.363390488754789232e+01 -8.482148111604411511e+00 1.454475225632749780e+00 5.549058692455690789e+01 -5.916661996826968561e+01 3.978439650286785678e+01 2.206082913051534433e+01 -1.302524774568759725e+01
-4.101699003411867750e+01 -1.842134180118595088e+00 5.336555180880862004e+00 -1.469418736137630255e+01 -1.815523935606591621e+01 -4.034337098293227086e+01 5.841935998468321714e+02 -1.681503497203259556e+01 -6.217118702792710394e+00 -1.875735971089100573e+01 -3.690450254912686745e+01 -5.813951251176727197e+01 -1.235434349335748649e+01 -2.328369243733704685e+00 1.276267983332951550e-01 1.119455862272325319e+01 -1.672338337963095967e+01 8.631939229108445133e+00 4.828318190010650213e+00 -4.803203139908840846e+00
-5.747012584607784191e+01 8.784021449319268982e+00 2.687946268478824763e+01 -2.027035450050597021e+01 -1.758079277909715188e+01 -6.133728485575397826e+01 -1.681503497203259556e+01 2.654362090965360608e+02 -1.699118942471849802e+00 -1.226186937686784262e+01 -4.439650281716689051e+01 -9.438253249283116020e+01 -1.126236318197124042e+01 -2.950112444583498750e+00 3.327625228728741558e+00 6.124469311522459947e+01 -1.557107282015066829e+01 3.761545519022216411e+01 2.042364095076887409e+01 8.223858446006786949e+00
-1.258519486749578498e+01 1.913451408033146706e+01 3.526840277210941110e+01 -3.957204716112934229e+00 8.048510627347919666e+00 -2.071765595501260648e+01 -6.217118702792712170e+00 -1.699118942471851579e+00 -8.575353082301711538e+01 1.854568666204047389e+01 1.350048583989411632e+00 -4.023647526677337538e+01 6.692098173347925893e+00 -1.731967344183004964e-01 5.497153222114254945e+00 8.240587429811988329e+01 8.493918855939229573e+00 4.688545462771074313e+01 2.515656384382386435e+01 2.444642644863633762e+01
-4.782374204528094452e+01 3.966086305760948960e+01 7.759713262590179284e+01 -1.596245606488101387e+01 7.735608092681061976e+00 -6.473648773252223521e+01 -1.875735971089100573e+01 -1.226186937686784972e+01 1.854568666204047389e+01 1.970378790643453470e+01 -1.613203308507952016e+01 -1.153221066832760329e+02 7.843009738158016830e+00 -1.566143800182631995e+00 1.173229336796167033e+01 1.806523863203348412e+02 9.417969820556795213e+00 1.039483307800458363e+02 5.587517697971669151e+01 4.941006005117635880e+01
-1.220851252727313039e+02 2.669867825201457023e+01 7.082396683255424819e+01 -4.283578585356517010e+01 -3.178995946386082849e+01 -1.337030526133585511e+02 -3.690450254912686745e+01 -4.439650281716689051e+01 1.350048583989411632e+00 -1.613203308507951306e+01 1.398589275284356290e+02 -2.096384633422140382e+02 -1.964735349444841717e+01 -6.046147328933960452e+00 9.296113891474250934e+00 1.623280714867570680e+02 -2.751818626297081494e+01 9.795849065990847748e+01 5.304715980502989225e+01 2.804699887576090944e+01
-2.260831179294159483e+02 -1.982703997041476285e+01 1.290068040521131465e+01 -8.126420990153069113e+01 -1.067580301778642990e+02 -2.182753366558432049e+02 -5.813951251176727908e+01 -9.438253249283116020e+01 -4.023647526677337538e+01 -1.153221066832760329e+02 -2.096384633422140382e+02 4.055366570093063388e+02 -7.324381162554089997e+01 -1.309958404381935893e+01 -1.976609053315249387e+00 2.292595616610623210e+01 -9.886871073208514815e+01 2.585672203380524081e+01 1.498802931082461143e+01 -3.920274598615063866e+01
-3.589850171074236584e+01 1.805396881141144405e+01 3.824439568888663388e+01 -1.231004996488574932e+01 -2.293761795812063298e+00 -4.363390488754787810e+01 -1.235434349335748649e+01 -1.126236318197124397e+01 6.692098173347925893e+00 7.843009738158018607e+00 -1.964735349444841717e+01 -7.324381162554091418e+01 7.665228378773402085e+01 -1.497519844484000995e+00 5.560409732370209746e+00 8.863432426442113865e+01 -1.034523153345969781e+00 5.171644251540939763e+01 2.786061946148059576e+01 2.167228255049276697e+01
-7.908401864627394140e+00 1.307010185173683503e+00 3.866585077563996720e+00 -2.786631529547718866e+00 -2.351350146382028772e+00 -8.482148111604411511e+00 -2.328369243733704685e+00 -2.950112444583499638e+00 -1.731967344183004687e-01 -1.566143800182632440e+00 -6.046147328933960452e+00 -1.309958404381935893e+01 -1.497519844484000995e+00 2.745967374691368832e+02 4.851321175839432009e-01 8.821676415434476226e+00 -2.074764465119578460e+00 5.396849447526419041e+00 2.928552368742548317e+00 1.260950663475877587e+00
4.805353425506697640e+00 7.945232914620116382e+00 1.257030230621269595e+01 1.937846753166918123e+00 7.470576060251922890e+00 1.454475225632751556e+00 1.276267983332948219e-01 3.327625228728742446e+00 5.497153222114254945e+00 1.173229336796167388e+01 9.296113891474250934e+00 -1.976609053315250719e+00 5.560409732370209746e+00 4.851321175839434230e-01 -4.738734455240488046e+02 2.967370394062129080e+01 7.305209719396514423e+00 1.634559083446665184e+01 8.723534078155699589e+00 1.073280411170944504e+01
1.030098176648417621e+02 1.128855664933924743e+02 1.714158318093651303e+02 3.993306823721567866e+01 1.204380623061663300e+02 5.549058692455690789e+01 1.119455862272326030e+01 6.124469311522459947e+01 8.240587429811988329e+01 1.806523863203348412e+02 1.623280714867570396e+02 2.292595616610621789e+01 8.863432426442113865e+01 8.821676415434474450e+00 2.967370394062129080e+01 -2.501310694169860653e+02 1.168754849204969162e+02 2.214243817561404626e+02 1.179799086154487355e+02 1.545062761869267263e+02
-4.904375289220832457e+01 2.361353581390678258e+01 5.045372622813156482e+01 -1.684715153795044174e+01 -3.860557029686392916e+00 -5.916661996826967140e+01 -1.672338337963095967e+01 -1.557107282015067362e+01 8.493918855939227797e+00 9.417969820556788108e+00 -2.751818626297081494e+01 -9.886871073208517657e+01 -1.034523153345969337e+00 -2.074764465119578460e+00 7.305209719396514423e+00 1.168754849204969162e+02 8.285946328400362404e+01 6.829288944679319684e+01 3.679899780899771855e+01 2.822480256943479660e+01
6.615952122195517404e+01 6.271107053896311356e+01 9.337876022171904822e+01 2.537352310098676966e+01 7.058402417947785068e+01 3.978439650286786389e+01 8.631939229108445133e+00 3.761545519022216411e+01 4.688545462771073602e+01 1.039483307800458363e+02 9.795849065990849169e+01 2.585672203380524081e+01 5.171644251540939763e+01 5.396849447526419041e+00 1.634559083446665184e+01 2.214243817561404626e+02 6.829288944679319684e+01 -6.437739673168042600e+02 6.400712343174073737e+01 8.635083293809304905e+01
3.615497669499554689e+01 3.351598550397287113e+01 4.974177081927150113e+01 1.384505174727032184e+01 3.805129393576854824e+01 2.206082913051535144e+01 4.828318190010646660e+00 2.042364095076887054e+01 2.515656384382386435e+01 5.587517697971669151e+01 5.304715980502988515e+01 1.498802931082461143e+01 2.786061946148059576e+01 2.928552368742548317e+00 8.723534078155699589e+00 1.179799086154487071e+02 3.679899780899771145e+01 6.400712343174073737e+01 -7.499279966213291573e+02 4.619645057890168260e+01
3.446143441964952814e+00 3.893177216325346279e+01 6.575096023479690643e+01 2.319936547352007494e+00 2.833314830332179213e+01 -1.302524774568759725e+01 -4.803203139908844399e+00 8.223858446006786949e+00 2.444642644863634118e+01 4.941006005117635169e+01 2.804699887576089878e+01 -3.920274598615064576e+01 2.167228255049277053e+01 1.260950663475878697e+00 1.073280411170944681e+01 1.545062761869267263e+02 2.822480256943480370e+01 8.635083293809304905e+01 4.619645057890168971e+01 -2.225750912869377771e+02
