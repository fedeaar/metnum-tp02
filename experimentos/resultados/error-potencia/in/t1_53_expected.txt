-7.840000000000000000e+02
-7.640000000000000000e+02
7.150000000000000000e+02
-6.880000000000000000e+02
-6.560000000000000000e+02
5.950000000000000000e+02
5.410000000000000000e+02
-4.760000000000000000e+02
3.950000000000000000e+02
3.730000000000000000e+02
-3.500000000000000000e+02
2.870000000000000000e+02
2.750000000000000000e+02
-2.740000000000000000e+02
2.290000000000000000e+02
1.010000000000000000e+02
-9.600000000000000000e+01
8.500000000000000000e+01
7.700000000000000000e+01
-1.000000000000000000e+01
